src/styles/main.scss

```diff
@@ -1,3 +1,35 @@
-body {
-  margin: 0;
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./utils/reset";
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.moyo__catalog {
+  margin: 0 auto;
+}
+
+.is-active {
+  color: $linkHoverTextColor;
+
+  &::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: $linkHoverTextColor;
+    border-radius: 8px;
+  }
```

Comment:
why do you move `.is-active` class outside nav?

Reply:
I thought that class="is-active" musn't be a part of my nav block.

I will return it back.